### ISSUE TITLE
Migrating modules fails

### ISSUE THREAD

**Issue by mahagr** [fastslack/jUpgrade#12]
I'm getting following error:

```
<br /><b>jos-Error</b>: Duplicate entry '90000053-0' for key 'PRIMARY' SQL=INSERT INTO `j25_modules_menu` (`moduleid`,`menuid`) VALUES ('90000053','0')<br />
Duplicate entry '90000053-0' for key 'PRIMARY' SQL=INSERT INTO `j25_modules_menu` (`moduleid`,`menuid`) VALUES ('90000053','0');|;5;|;modules
```

**Comment by fastslack** [comment]
This error is fixed: 04977d632c1e1741aece944acbcd9b4817001b23

**Comment by mahagr** [comment]
Your change seems to be working, but I'm not quite sure what it does. :)
